### PR TITLE
try to repeat comment output in plan

### PR DESCRIPTION
**Pull request by pseudonative**

### CODE REVIEW

File: terraform/ec2-instance.tf [hunk 1]
```diff
@@ -9,7 +9,7 @@ resource "aws_instance" "myec2vm" {
     aws_security_group.vpc-ssh.id,
     aws_security_group.vpc-web.id
   ]
-  count = 1
+  count = 2
```
Comment (shoshone-dev) [inline terraform/ec2-instance.tf]: set the count to 1 and the type to qa

**Review by shoshone-dev — CHANGES_REQUESTED** [review]

File: terraform/ec2-instance.tf (hunk 1 above)
Comment (pseudonative) [reply terraform/ec2-instance.tf]: fixed

**Review by shoshone-dev — APPROVED** [review]
/lgtm_dev

**Pull request merged**